src/script/pages/app-testing.ts

```diff
@@ -12,75 +12,79 @@ import { smallBreakPoint } from '../utils/breakpoints';
 import { runAllTests } from '../services/tests';
 import '../components/app-header';
 
+//@ts-ignore
+import style from '../../../styles/animations.css';
+
 @customElement('app-testing')
 export class AppTesting extends LitElement {
   @internalProperty() loading = false;
-  @internalProperty() currentPhrase: string = "PWABuilder is loading your PWA in the background";
+  @internalProperty() currentPhrase: string =
+    'PWABuilder is loading your PWA in the background';
 
   static get styles() {
-    return css`
-      :host {
-        display: flex;
-        flex-direction: column;
-        background: url(/assets/images/background-copy.webp);
-        height: 100vh;
-        overflow: hidden;
-      }
-
-      #testing-container {
-        flex: 0.7;
-        display: flex;
-        flex-direction: column;
-        align-items: center;
-        justify-content: center;
-
-        text-align: center;
+    return [
+      style,
```

Comment:
Is this how we use global styles in Lithtml?

Reply:
Yep, this would be the way to do it going forward! So for your header styles for instance, you can put them in a 'header.styles.css' and then import that wherever you want